docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters-free.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Create a free cluster
+linkTitle: Create a free cluster
+description: Create free clusters in Yugabyte Cloud.
+headcontent:
+image: /images/section_icons/deploy/enterprise.png
+menu:
+  latest:
+    identifier: create-clusters-free
+    parent: cloud-basics
+    weight: 40
+isTocNested: true
+showAsideToc: true
+---
+
+Use a free cluster to get started with YugabyteDB. Although not suitable for production workloads, a free cluster includes enough resources to start exploring the core features available for developing applications with YugabyteDB.
+
+## Features
+
+Your free cluster includes the following:
+
+- Single node.
+- Up to 2 vCPUs and 10 GB of storage, depending on the cloud provider.
+- An [edge release](../../cloud-faq/#what-version-of-yugabytedb-does-my-cluster-run-on) of YugabyteDB, which provides early access to new features.
+- Available in all [regions](../../release-notes#cloud-provider-regions).
+
+Share your feedback, questions, and suggestions with other users on the [YugabyteDB Community Slack](https://www.yugabyte.com/slack).
+
+## Creating a free cluster
+
+To create a free cluster, on the **Clusters** page, click **Add Cluster** to start the **Create Cluster** wizard.
+
+The **Create Cluster** wizard has the following pages:
+
+1. [Select Cluster Type](#select-cluster-type)
+1. [Cluster Settings](#cluster-settings)
+1. [Database Admin Credentials](#database-admin-credentials)
+
+### Select Cluster Type
+
+![Add Cluster Wizard - Select Type](/images/yb-cloud/cloud-addcluster1-type.png)
+
+Select **Yugabyte Cloud Free** and click **Next** to display the **Cluster Settings** page.
+
+### Cluster Settings
+
+![Add Cluster Wizard - Cluster Settings](/images/yb-cloud/cloud-addcluster-free2.png)
+
+Set the following options:
+
+- **Provider**: Choose a cloud provider - AWS or GCP.
+- **Cluster Name**: Enter a name for the cluster.
+- **Region**: Choose the Region where the cluster will be located.
+
+### Database Admin Credentials
+
+The admin credentials are required to connect to the YugabyteDB database that is installed on the cluster. (You can [add additional users](../../cloud-secure-clusters/add-users/) after the cluster is provisioned.)
+
+![Add Cluster Wizard - Admin Settings](/images/yb-cloud/cloud-addcluster-admin.png)
+
+You can use the default credentials generated by Yugabyte Cloud, or add your own.
+
+Download the credentials, and click **Create Cluster**.
+
+{{< warning title="Important" >}}
+
+Save your database credentials in a safe place. If you lose these credentials, you won't be able to use the database.
```

Comment:
Suggestion:
Save your database credentials. If you lose them, you will not be able to use the database.